lib/structures/Member.js

```diff
@@ -57,7 +57,11 @@ class Member extends Base {
                 throw new Error("User associated with Member not found: " + data.id);
             }
         } else if(data.user) {
-            this.user = new User(data.user, client);
+            if(!client) {
+                this.user = new User(data.user, client);
```

Comment:
Useless client

Reply:
Oh okay, i just thought that the client property may not always be passed.